components/usage-api/typescript/src/usage/v1/usage_grpc_pb.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Is it necessary to commit both the generated `.ts` and `.js` files?

Reply:
Yes. The JS provides the implementation, while the TS is only the type-def such that it type-checks nicely.